modules/nf-core/paml/codeml/tests/main.nf.test

```diff
@@ -0,0 +1,71 @@
+nextflow_process {
+
+    name "Test Process PAML_CODEML"
+    script "../main.nf"
+    process "PAML_CODEML"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "paml"
+    tag "paml/codeml"
+
+    test("complete config case_a") {
```

Comment:
Can you describe the test name a little bit more? Such as including the data type `(phy, tree, ctil)`. In that way, when you create an additional test case, we can know if it uses another type of input data.